homeassistant/components/utility_meter/sensor.py

```diff
@@ -355,6 +355,7 @@ class UtilityMeterSensor(RestoreSensor):
 
     _attr_translation_key = "utility_meter"
     _attr_should_poll = False
+    _attr_force_update = True
```

Comment:
This is a really bad pattern; it causes unneeded updates and makes our history grow fast (including our databases).

We have basically removed it everywhere over the past years. For the use case of seeing if an integration entity is still update, we have a reported timestamp available on the entities (that are updated when they report in, even if it means there is no state change).


I'm going to friendly decline this change for the above reason.

../Frenck

Reply:
in that case should I make utility_meter to listen and passthrough state_reported events? because right now it doesn't do that, so it would completely swallow them